.github/workflows/maven-check-deps-build-publish.yml

```diff
@@ -43,7 +43,10 @@ jobs:
 
       - name: Check for dependency changes
         id: compareDependencyChanges
-        run: bash ${GITHUB_WORKSPACE}/.github/scripts/compare-dependency-reports.sh
+        run: |
+          curl -O https://raw.githubusercontent.com/mekomsolutions/shared-github-workflow/main/.github/scripts/compare-dependency-reports.sh
```

Comment:
Isn't there any variable available that contains this repo name/location: `mekomsolutions/shared-github-workflow` ?
It would be ideal to avoid hardcoding the name/location.

Reply:
> Isn't there any variable available that contains this repo name/location: mekomsolutions/shared-github-workflow ?

Yes. `${GITHUB_REPOSITORY}`
However, we can't use this because it will reference the repository running the workflow.